### PR TITLE
Release

### PR DESCRIPTION
**Pull request by xqxpx**
Minor bug fixes: see commit descriptions

### CODE REVIEW

File: src/backend/dev_helpers.rs [hunk 1]
```diff
@@ -62,9 +62,28 @@ async fn clear_buckets() {
 #[update]
 fn replace_user_principal(principal: String, user_id: UserId) {
```
Comment (aligatorr89) [inline src/backend/dev_helpers.rs:63]: I don't see this method being exposed anywhere?

**Review by aligatorr89 — APPROVED** [review]

File: src/backend/env/mod.rs [hunk 2]
```diff
@@ -271,6 +271,17 @@ pub enum Destination {
 }
 
 impl State {
+    pub fn register_post_tags(&mut self, post_id: PostId, tags: &BTreeSet<String>) {
+        for tag in tags {
```
Comment (aligatorr89) [inline src/backend/env/mod.rs:275]: It's missing removing tags. If i remove tag from post, it still shows it when searching for it

**Review by aligatorr89 — CHANGES_REQUESTED** [review]

File: src/backend/dev_helpers.rs (hunk 1 above)
Comment (chmllr) [reply src/backend/dev_helpers.rs:63]: I think those are helper methods — see the file name.

**Review by chmllr — APPROVED** [review]
LGTM

**Review by aodl — COMMENTED** [review]
Why move post archiving to the daily routine? It looks like the hourly routine tends to take more than a few minutes. Presumably this change is intended to bring that down to less than a minute to reduce noise in the hourly logs? Would an alternative approach be to increase the threshold for hourly logs slightly?

**Comment by xqxpx** [comment]
@aodl right it is to decrease the verbosity. Also it takes less than 0B instructions, so there is no danger in moving it to daily for now. I used these logs to watch the stability, but see no issues anymore, so this procedure is simply not really interesting.

File: src/backend/env/mod.rs (hunk 2 above)
Comment (taggrx) [reply src/backend/env/mod.rs:275]: That's fine, the deleted posts are not rendered in most feed anyway. also I think it's a good to see that the post existed. However, also note that tag indexes are rotating. They only keep the last 1k posts. So any de;eted post will disappear eventually anyway, this is why I see no reasons to complicate the code.

Comment (aodl) [reply src/backend/env/mod.rs:275]: But what about posts that are not deleted, but are edited to remove erroneously added tags? I think this is what @aligatorr89 was describing

Comment (aligatorr89) [reply src/backend/env/mod.rs:275]: @taggrx , I described removing tags from post (edit post)

**Pull request merged**